<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBranchDialog</class>
 <widget class="QDialog" name="NewBranchDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New branch</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="optionsLabel">
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="switchToBranchCheckBox">
     <property name="text">
      <string>&amp;Switch to branch after creating</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="upstreamLayout">
     <item>
      <widget class="QCheckBox" name="upstreamCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Track upstream branch:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="upstreamComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertPolicy::NoInsert</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="recurseSubmodulesCheckBox">
     <property name="text">
      <string>â€¦then update submodules recursively</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>switchToBranchCheckBox</tabstop>
  <tabstop>recurseSubmodulesCheckBox</tabstop>
  <tabstop>upstreamCheckBox</tabstop>
  <tabstop>upstreamComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewBranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewBranchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upstreamCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>upstreamComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>switchToBranchCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>recurseSubmodulesCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
